db/migrate/20161103101916_create_applications.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :applications do |t|
       t.string :motivation
       t.references :user, index: true, foreign_key: true
-      t.references :workshop, index: true, foreign_key: true
+      t.references :event, index: true, foreign_key: true
```

Comment:
You should never touch old migration files (afaik). This is the old file that created the workshop column that later was renamed to event. I don't know what happens if you mess with this file and we need to execute it again. Please remove this change.